.kilocode/rules/页面的定义.md

```
@@ -0,0 +1,13 @@
+# 页面的定义.md
+
+规范页面的定义
+
+## 指导原则
+
+- 产品页 指的是 前台 /product/{产品名}/，是wordpress的产品页。
+    - 样式文件在 public\modules\class-pw-cdn-loader.php 中引用
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)